skins/oasis/modules/ContributeMenuController.class.php

```diff
@@ -36,6 +36,15 @@ public function executeIndex() {
 				'href' =>  $specialPageTitle->getLocalURL(),
 			];
 
+			/**
+			 * id for flow tracking
+			 * @see https://wikia-inc.atlassian.net/browse/WW-343
+			 */
+			if ( $specialPageName === 'CreatePage' ) {
+				$attrs[ 'id' ] = 'contribute-button-create-page';
+				$attrs[ 'href' ] = $attrs[ 'href' ] . '?flow=create-page-contribute-button';
```

Comment:
Time to think about reusing the consts in both front- and back-end - this approach will be error prone in long run.
Please add a todo here